src/js/components/DataTable/Body.js

```diff
@@ -45,24 +45,24 @@ const Row = memo(
         size={size}
         active={active}
         aria-disabled={(onClickRow && isDisabled) || undefined}
-        onClick={
-          onClickRow && !isDisabled
-            ? (event) => {
-                // extract from React's synthetic event pool
-                event.persist();
-                const adjustedEvent = event;
-                adjustedEvent.datum = datum;
-                adjustedEvent.index = index;
-                onClickRow(adjustedEvent);
-              }
-            : undefined
-        }
+        onClick={(event) => {
```

Comment:
Can we only bind a handler if needed? For example: `onClick={(onClickRow && !isDisabled) ? (event) => {} : undefined}`

Reply:
done